cmd/konvoy-image-wrapper/cmd/wrapper.go

```diff
@@ -95,11 +98,23 @@ func NewRunner() *Runner {
 	if err != nil {
 		log.Fatalf("error getting user home directory: %v", err)
 	}
+	var containerEngine string
+	switch p := os.Getenv(containerEngineEnv); p {
+	case "":
+		containerEngine = detectContainerEngine()
+	case "podman":
+		containerEngine = containerEnginePodman
+	case "docker":
+		containerEngine = containerEngineDocker
+	default:
+		log.Printf("ignoring unknown value %q for %s", p, containerEngineEnv)
```

Comment:
not sure. if we should throw error here when user specifies unknown container engine.

Reply:
throwing an error is good i think